client/python/test/test_client.py

```diff
@@ -85,6 +88,13 @@ def test_raise_error_invalid_state(self):
         with pytest.raises(ValueError, match="invalid state"):
             Client("gradio-tests/paused-space")
 
+    def test_raise_error_max_file_size(self, max_file_size_demo):
+        with connect(max_file_size_demo, max_file_size="1b") as client:
+            with pytest.raises(
+                ValueError, match=f"File {__file__} exceeds the maximum file size"
+            ):
+                client.predict(file(__file__), api_name="/upload_1b")
```

Comment:
Is this tryping to upload the test file itself? That's interesting but maybe more robust to upload a sample test file that we know is of a fixed size. We should also test that files that are below the specified upload limit are successfully uploaded

Reply:
Yea good point!